client/trino-client/src/main/java/io/trino/client/ClientStandardTypes.java

```diff
@@ -40,11 +43,14 @@ public final class ClientStandardTypes
     public static final String ARRAY = "array";
     public static final String MAP = "map";
     public static final String JSON = "json";
+    public static final String JSON_2016 = "json2016";
     public static final String IPADDRESS = "ipaddress";
     public static final String UUID = "uuid";
     public static final String GEOMETRY = "Geometry";
     public static final String SPHERICAL_GEOGRAPHY = "SphericalGeography";
     public static final String BING_TILE = "BingTile";
+    public static final String KDB_TREE = "KdbTree";
```

Comment:
Same with this one. This type is just there to support some types of geospatial joins